Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/SurfaceMagnetism.cs

```diff
@@ -77,28 +77,28 @@ public LayerMask[] MagneticSurfaces
 
         [SerializeField]
         [Tooltip("Max distance for raycast to check for surfaces")]
-        private float maxDistance = 3.0f;
+        private float maxRaycastDistance = 50.0f;
 
         /// <summary>
         /// Max distance for raycast to check for surfaces
         /// </summary>
-        public float MaxDistance
+        public float MaxRaycastDistance
         {
-            get { return maxDistance; }
-            set { maxDistance = value; }
+            get { return maxRaycastDistance; }
+            set { maxRaycastDistance = value; }
         }
 
         [SerializeField]
         [Tooltip("Closest distance to bring object")]
```

Comment:
Please using FormerlySerializedAs to avoid breaking previous users of this script

Reply:
gah didn't know FormerlySerializedAs existed! so useful! thanks